otx/algorithms/common/utils/utils.py

```diff
@@ -16,13 +16,22 @@
 
 import importlib
 import inspect
+from collections import defaultdict
 from typing import Callable, Optional, Tuple
 
 import yaml
 
 from otx.api.utils.argument_checks import YamlFilePathCheck, check_input_parameters_type
 
 
+class UncopiableDefaultDict(defaultdict):
+    """Uncopiable defaultdict."""
+
+    def __deepcopy__(self, memo):
+        """Deepcopy."""
```

Comment:
Could you update docstring more specifically? I believe it's hard to understand for other contributors.

Reply:
If you have some, please give and I will update. I think it's enough to understand its functionality as other methods in this file.

Reply:
I agree with Jihwan's opinion. The purpose of the docstring is to communicate what the function does. The current annotations make it difficult to understand the functionality.
It's hard to understand the meaning of `__deepcopy__` in `UncopiableDefaultDict` without looking deeply.

Reply:
Then, could you please give some idea to update? I can't brainstorm more than "defauldict type object to avoid deepcopy."

Reply:
I think your comment ("Defauldict type object to avoid deepcopy.") is enough. :)
or `A function that returns itself to avoid deepcopy.`

Reply:
I fixed it. Thanks.